packages/testing/src/api/index.ts

```diff
@@ -66,7 +70,7 @@ function createUnsecuredJwt(auth: object): string {
   ].join('.');
 }
 
-export function apps(): (FirebaseApp | null)[] {
```

Comment:
is this no longer capable of returning null?

Reply:
Correct. I have no idea why this was ever returning an array of potentially null objects. That was pretty dangerous, since we were calling `.delete()` on the elements of this array, so those calls would throw exceptions if the element were `null`.